uiplib/UIP.py

```diff
@@ -9,7 +9,7 @@
 
 def main():
     settingsParser = ParseSettings()
-    settings = settingsParser.settings
+    settings = settingsParser.get_settings(sys.argv[1:])
```

Comment:
Yeah, I'm making a few changes to faciliate `__init__` and correspond with the tests too.